controllers/topics-controllers

```diff
@@ -0,0 +1,19 @@
+const { selectTopics } = require('../models/topics.models.js')
+
+
+const getTopics = (req, res, next) => {
+    console.log('in controller 1')
```

Comment:
Remove console logs before you submit your code. As we discussed, get them in there to track your data for sure, but when you know it's working, get them gone